aries-site/src/examples/templates/forms/FormHeaderExample.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
```

Comment:
This is more of a question for @gregfuruiye, but do we need / should we have a place in the design which specifically tells users "* indicates a required field" or is this omitted because it is well understood?

Reply:
Did this question get asked? @britt6612 @gregfuruiye

Reply:
I think there should be content around the * label 
right now we have this so maybe just adding a few sentences 
https://design-system.hpe.design/templates/forms#required-vs-optional-fields